vmdb/app/controllers/application_controller.rb

```diff
@@ -966,8 +966,8 @@ def get_reports_menu(group=session[:group], tree_type="reports", mode="menu")
     rec = MiqGroup.find_by_id(group)
     user = current_user
     @sb[:grp_title] = user.admin_user? ?
-      "#{session[:customer_name]} (#{_("All %s") % ui_lookup(:models=>"MiqGroup")})" :
-      "#{session[:customer_name]} (#{_("%s") % "#{ui_lookup(:model=>"MiqGroup")}: #{user.current_group.description}"})"
+      "#{current_brand.customer_name} (#{_("All %s") % ui_lookup(:models=>"MiqGroup")})" :
+      "#{current_brand.customer_name} (#{_("%s") % "#{ui_lookup(:model=>"MiqGroup")}: #{user.current_group.description}"})"
```

Comment:
This method name feels weird.  When I think Brand I think logo, but then this has customer name.  Although I understand that's part of brand, I more associate that with a tenant.  Wondering if this should just say `current_tenant`.  I'm thinking that a tenant will expose it's brand anyway, so even `current_tenant.logo` makes sense.  @gmcculloug Thoughts?